src/tools/tidy/src/deps.rs

```diff
@@ -143,6 +143,8 @@ const PERMITTED_RUSTC_DEPENDENCIES: &[&str] = &[
     "elsa",
     "ena",
     "equivalent",
+    "errno",
+    "errno-dragonfly",
```

Comment:
Oh, probably that's why it appears here https://github.com/rust-lang/rust/pull/113449/files#diff-1b98ff2ba41d961103aadb79662434330725d9503ca09496820a171d37390a95